presto-main/src/main/java/com/facebook/presto/operator/HashBuilderOperator.java

```diff
@@ -337,13 +338,17 @@ private void updateIndex(Page page)
     {
         index.addPage(page);
 
+        // Why not index.getEstimatedSize.toBytes()? Because we are adding a calculation for the expected size of the PagesHash we will build soon
+        int numPositions = index.getPositionCount();
+        long estimatedPagesHashSize = calculatedPredictedSizeDelta(numPositions);
```

Comment:
`s/calculatedPredictedSizeDelta/PageHash.getEstimatedSize(numPositions)`?

Reply:
The reason to have a separate function here is for testing visibility. Do you disagree?

Reply:
(y)